pipeline-templates/job/codebuild.yml

```diff
@@ -0,0 +1,50 @@
+parameters:
```

Comment:
Is the file name correct? I believe all other repos have the code-build

Reply:
lets follow one naming convention